app/src/main/java/pm/gnosis/heimdall/ui/safe/add/AddSafeViewModel.kt

```diff
@@ -29,6 +29,7 @@ import pm.gnosis.utils.exceptions.InvalidAddressException
 import pm.gnosis.utils.hexAsBigInteger
 import retrofit2.HttpException
 import timber.log.Timber
+import java.math.BigInteger
 import javax.inject.Inject
 
 class AddSafeViewModel @Inject constructor(
```

Comment:
Can we clean up everything related to "addExistingSafe", as we won'T need that anymore, right?